tools/debugger/tests/do_finish.expected

```diff
@@ -0,0 +1,19 @@
+Connecting to: localhost:6501
+Connection created!!!
+Stopped at tools/debugger/tests/do_finish.js:20
+(escargot-debugger) b do_finish.js:24
+Breakpoint 1 at tools/debugger/tests/do_finish.js:24 (in f() at line:22, col:1)
+(escargot-debugger) c
+Stopped at breakpoint:1 tools/debugger/tests/do_finish.js:24 (in f() at line:22, col:1)
+(escargot-debugger) finish
+Stopped at tools/debugger/tests/do_finish.js:32 (in g() at line:29, col:1)
+(escargot-debugger) f
+Stopped at tools/debugger/tests/do_finish.js:38
+(escargot-debugger) b do_finish.js:41
+Breakpoint 2 at tools/debugger/tests/do_finish.js:41 (in gen() at line:40, col:1)
+(escargot-debugger) f
+Stopped at breakpoint:2 tools/debugger/tests/do_finish.js:41 (in gen() at line:40, col:1)
```

Comment:
I have one more question.
For the above case, when "finish" is executed on the global code debugger runs the program until the next breakpoint encountered inside `gen()` instead of finishing the program.
But from the [description](https://sourceware.org/gdb/onlinedocs/gdb/Continuing-and-Stepping.html), "finish" is defined as follow.
> finish : Continue running until just after function in the selected stack frame returns.

I'm wondering what is the correct operation when another breakpoint is met during the execution of "finish" command. Should we stop at the breakpoint or ignore it?

Reply:
In my experience breakpoints always takes priority over anything. So if a breakpoint is encounterd the execution stops regardless the last command.

Reply:
Note: we can do this differently of course, but this is my experience with debuggers.

Reply:
Your answer makes everything clear! Thanks.